src/main/java/edu/berkeley/ground/db/CassandraClient.java

```diff
@@ -192,6 +276,63 @@ public void delete(List<DbDataContainer> predicates, String table) {
     this.session.execute(statement);
   }
 
+	/**
+   * Deletes a column from a table (sets column to null)
```

Comment:
Deleting a column from a table sounds very different from changing it to null. I think `setColumnValueToNull` might be a more apt name?